com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/InlineValueHandler.java

```diff
@@ -166,7 +166,7 @@ private static IMethod findMethodInLocalTypes(IMethod enclosingMethod, int stopp
      */
     private static Position getPosition(IBuffer buffer, int offset) {
         int[] result = JsonRpcHelpers.toLine(buffer, offset);
-        if (result == null && result.length < 1) {
```

Comment:
compiler check:  if `result` is null,then `result.lenth` will throw NullPointerException